suggester/src/main/java/org/opengrok/suggest/Suggester.java

```diff
@@ -143,6 +151,16 @@ public Suggester(
         this.allowedFields = new HashSet<>(allowedFields);
         this.timeThreshold = timeThreshold;
         this.rebuildParallelismLevel = rebuildParallelismLevel;
+
+        suggesterRebuildCounter = Counter.builder("suggester.rebuild").
+                description("suggester rebuild count").
+                register(registry);
+        suggesterRebuildTimer = Timer.builder("suggester.rebuild.latency").
```

Comment:
Now that I think about it. Does not `Timer` include also a `Counter` by default? In the code it seems they do: https://github.com/micrometer-metrics/micrometer/blob/master/implementations/micrometer-registry-prometheus/src/main/java/io/micrometer/prometheus/PrometheusTimer.java

Could you please check if you can plot it in Grafana?

Reply:
Indeed it does. I wanted to mention it previously that I'd like to keep it separate.

```
$ curl -X GET -s http://grok-stage.cz.oracle.com/source/metrics/prometheus | grep ^suggest
suggester_init_latency_seconds_count 1.0
suggester_init_latency_seconds_sum 0.035
suggester_init_latency_seconds_max 0.0
suggester_rebuild_latency_seconds_max 0.0
suggester_rebuild_latency_seconds_count 30.0
suggester_rebuild_latency_seconds_sum 0.012
suggester_rebuild_total 30.0
```

Reply:
Okay, no problem :)